src/account/AccountPickScreen.js

```diff
@@ -56,7 +56,7 @@ class AccountPickScreen extends PureComponent<Props> {
             text="Add new account"
             style={styles.button}
             onPress={() => {
-              dispatch(navigateToRealmScreen(''));
+              dispatch(navigateToRealmScreen());
```

Comment:
Good, we don't need that indeed, as we make sure to initialize the variable with '' if missing in `RealmScreen`.